dbms/src/Flash/ResourceControl/LocalAdmissionController.cpp

```diff
@@ -167,23 +167,41 @@ std::optional<LocalAdmissionController::AcquireTokenInfo> LocalAdmissionControll
     const ResourceGroupPtr & resource_group,
     bool is_periodically_fetch)
 {
-    double token_consumption = resource_group->getAndCleanConsumptionDelta();
+    double token_consumption = 0.0;
     double acquire_tokens = 0.0;
+    const auto now = std::chrono::steady_clock::now();
```

Comment:
can use `using SteadyClock = std::chrono::steady_clock;` like https://github.com/pingcap/tiflash/blob/a55a10fbd6e1d9588e0a1c63de224c76527261c5/dbms/src/Common/MPMCQueue.h#L273 to simplify the code

Reply:
done